packages/addons/service/docker/package.mk

```diff
@@ -29,6 +29,7 @@ PKG_SECTION="service/system"
 PKG_SHORTDESC="Docker is an open-source engine that automates the deployment of any application as a lightweight, portable, self-sufficient container that will run virtually anywhere."
 PKG_LONGDESC="Docker containers can encapsulate any payload, and will run consistently on and between virtually any server. The same container that a developer builds and tests on a laptop will run at scale, in production*, on VMs, bare-metal servers, OpenStack clusters, public instances, or combinations of the above."
 PKG_AUTORECONF="no"
+PKG_SOURCE_DIR="moby-$PKG_VERSION"
```

Comment:
@escalade:  you are right. i found it, while i upgrade to gcc7. actual, i had no time to split it into single branches.